src/versionbits.h

```diff
@@ -84,4 +86,18 @@ ThresholdState VersionBitsState(const CBlockIndex *pindexPrev,
     VersionBitsCache &cache);
 uint32_t VersionBitsMask(const Consensus::Params &params, Consensus::DeploymentPos pos);
 
+// tweak to configure and dynamically change what you are voting for
+extern std::string bip135Vote;
+
+// Set the specified feature to the vote.  Pass vote=-1 to check existence of feature
+// returns false if the feature is not defined.
+bool AssignBip135Vote(const std::string &feature, int vote = true);
+// Set all votes to false
```

Comment:
I think vote=true is a little confusing and maybe prone to operator error, I was thinking to set vote=-1 if no second param was set, and then return the current status so the operator can know whether it's on or not before trying to change it.  With the way it is now, I'm afraid someone may end up turning on a vote when they didn't intend to.